src/containerapp/azext_containerapp/custom.py

```diff
@@ -1093,7 +1105,7 @@ def containerapp_up(cmd,
     up_output(app, no_dockerfile=(source and not _has_dockerfile(source, dockerfile)))
 
 
-def containerapp_up_logic(cmd, resource_group_name, name, managed_env, image, env_vars, ingress, target_port, registry_server, registry_user, workload_profile_name, registry_pass):
+def containerapp_up_logic(cmd, resource_group_name, name, managed_env, image, env_vars, ingress, target_port, registry_server, registry_user, workload_profile_name, registry_pass, environment_type=None):
```

Comment:
managed_env here is for both managed and connected?

Reply:
Yes. It is for both managed and connected. Do you think we should change the argument name?